src/urh/dev/native/lib/cbladerf.pxd

```diff
@@ -95,15 +95,19 @@ cdef extern from "libbladeRF.h":
         int bladerf_get_bandwidth(bladerf *dev, bladerf_module module, unsigned int *bandwidth)
 
     IF BLADERF_API_VERSION >= 2:
+        int bladerf_set_frequency(bladerf *dev, bladerf_channel ch, bladerf_frequency frequency)
+        int bladerf_get_frequency(bladerf *dev, bladerf_channel ch, bladerf_frequency *frequency)
```

Comment:
This makes the CI [fail](https://dev.azure.com/jopohl/urh/_build/results?buildId=1358&view=logs&j=2e703997-b85c-5955-fe15-cdf93bef6757&t=b6fc0516-e52d-5bcf-4f04-ad375cf5260d&l=80). It might help to move the 
```cython
IF BLADERF_API_VERSION >= 2:
    ctypedef uint64_t bladerf_frequency
ELSE:
    ctypedef unsigned int bladerf_frequency
```
on top of this

Reply:
@jopohl, I needed to set this right at the top of file since with bladeRF 1.9.1 it wasn't detecting `bladerf_frequency`

Reply:
Nice, CI works again. :)